README.md

```diff
@@ -21,18 +35,59 @@ Just make sure your changes and pull requests are in accordance with the [contri
 
 We are not currently accepting translations of the game on our main repository. If you would like to translate the game into another language consider creating a fork or contributing to a fork.
 
+## Project activity
+<!---
+CHANGE THE API!!!! API can be obtained on the website repobeats.axiom.co | HERE, FOR EXAMPLE, THERE IS AN API FROM ANOTHER REPOSITORY
+--->
+![Project activity](https://repobeats.axiom.co/api/embed/4637fb51923408d570b8e555b3fde24eedb2bfea.svg 'Repobeats analytics image')
+
+## Project contributors
+
+List of people who contributed to the project:
+
+[![Contributors](https://contrib.rocks/image?repo=space-wizards/space-station-14)](https://github.com/space-wizards/space-station-14/graphs/contributors)
+
```

Comment:
This part takes a huge chunk of the page and honestly... it's really unnecessary information. I see this on a bunch of gh repos and it just kinda feels like filler.